service/matching/matchingEngine.go

```diff
@@ -707,6 +707,52 @@ func (e *matchingEngineImpl) listTaskQueuePartitions(request *matchingservice.Li
 	return partitionHostInfo, nil
 }
 
+func (e *matchingEngineImpl) UpdateWorkerBuildIdOrdering(
+	hCtx *handlerContext,
+	req *matchingservice.UpdateWorkerBuildIdOrderingRequest,
+) (*matchingservice.UpdateWorkerBuildIdOrderingResponse, error) {
+	namespaceID := namespace.ID(req.GetNamespaceId())
+	taskQueueName := req.GetRequest().GetTaskQueue()
+	taskQueue, err := newTaskQueueID(namespaceID, taskQueueName, enumspb.TASK_QUEUE_TYPE_WORKFLOW)
+	tqMgr, err := e.getTaskQueueManager(taskQueue, enumspb.TASK_QUEUE_KIND_NORMAL)
```

Comment:
just a note that if merging after #3033, this should change to getTaskQueueManagerInitialized (if this lands first I'll change it)